draft-ietf-jsonpath-base.md

```diff
@@ -259,7 +259,7 @@ Parameter:
 : Formal parameter that can take Arguments (actual parameters) in a function-expression.
 
 Normalized Path:
-: A simple form of JSONPath expression that identifies a node in a value by
+: A minimal form of JSONPath expression that identifies a node in a value by
```

Comment:
I'm not sure this is right either.  If anything, the "normalized" syntax is more verbose for many cases (e.g. `$.a.b` vs `$['a']['b']`.

Maybe "formal"?

Reply:
Not sure "formal" is a great deal better. Perhaps just skip the adjective altogether?
```suggestion
: A form of JSONPath expression that identifies a node in a value by
```

Reply:
> Not sure "formal" is a great deal better. Perhaps just skip the adjective altogether?

Uniquely?

Reply:
I think uniqueness of the identified node is implicit. Or did you mean uniqueness of the query??

Reply:
For each node, there is exactly one ("unique") normalized path.
That is the main point of them to me.

Reply:
Get rid of minimal now we have expanded the definition:
```suggestion
: A form of JSONPath expression that identifies a node in a value by
```